tcms/tests/__init__.py

```diff
@@ -227,3 +227,26 @@ def setUpTestData(cls):
         cls.execution_4 = executions[3]
         cls.execution_5 = executions[4]
         cls.execution_6 = executions[5]
+
+
+class AssignPermissionsToTestUser(LoggedInTestCase):
+    """Assign required permissions to user """
+
+    def setUpTestData(cls):
+        super(LoggedInTestCase, cls).setUpTestData()
+
+        cls.tester = LoggedInTestCase.tester
+
+    def remove_all_permissions_except_tested(self, tested_permission):
```

Comment:
the name is too long. you can shorten it to something like `remove_all_permissions_except`, because the signature of the method helps you, and you would be using it like `self.remove_all_permissions_except("permission_under_test")`